workers/generic-worker/gw-decision-task/tasks.yml

```diff
@@ -106,7 +106,7 @@ WorkerPools:
   proj-taskcluster/gw-ci-windows10-arm:
     Platform: 'Windows 10 (arm)'
     OS: 'windows'
-    # There is no arm release for go 1.13.7 on windows, but 386 release works
+    # There is no arm release for go 1.15.6 on windows, but 386 release works
```

Comment:
Confirmed that this is still the case on https://blog.golang.org/ports. There is [an open issue](https://github.com/golang/go/issues/36439) to track support for ARM64 Windows 10 systems.

Reply:
I think we may even run Windows/arm32 (see https://github.com/golang/go/issues/26148). I can check, I guess.